spec/models/base_organisation_spec.rb

```diff
@@ -56,5 +56,16 @@
       org.save
     end
   end
+  
+  describe 'friendly_id' do
+     
+     it 'should use SetupSlug service for setting slugs' do
+       service_double = class_double('SetupSlug').as_stubbed_const
```

Comment:
if we make this be 

```ruby 
service_double = class_double(SetupSlug).as_stubbed_const
```
then we'll get verifying double functionality:

https://relishapp.com/rspec/rspec-mocks/v/3-4/docs/verifying-doubles/using-a-class-double